website/addons/owncloud/utils.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from website.oauth.models import ExternalAccount
+from website.security import encrypt, decrypt
+
+
+class ExternalAccountConverter:
```

Comment:
It will certainly get in faster in a separate PR, but I don't know of any other addons nearing completion that use Basic Auth, so either is fine.